tests/unit/scripts/test_initialize_library.py

```diff
@@ -9,10 +9,10 @@
 def test_init_library():
     # Create the user agains the current mongo database
     with patch('pymongo.MongoClient') as MongoClient, \
-         patch('arctic.scripts.arctic_init_library.logger', autospec=True) as logger, \
-         patch('arctic.scripts.arctic_init_library.Arctic', autospec=True) as Arctic, \
-         patch('arctic.scripts.arctic_init_library.get_mongodb_uri', autospec=True) as get_mongodb_uri, \
-         patch('arctic.scripts.arctic_init_library.do_db_auth', autospec=True) as do_db_auth:
+         patch('arctic.scripts.arctic_init_library.logger') as logger, \
```

Comment:
Why no more `autospec` ?

Without autospec these tests don't really test antyhing - it's possible for the API to change and these wiring tests will happily pass...